SmartDeviceLink/SDLMenuManager.h

```diff
@@ -0,0 +1,31 @@
+//
+//  SDLMenuManager.h
+//  SmartDeviceLink
+//
+//  Created by Joel Fischer on 4/9/18.
+//  Copyright © 2018 smartdevicelink. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
```

Comment:
There a few nullability annotation warnings. 
Class is missing `NS_ASSUME_NONNULL_BEGIN` & `NS_ASSUME_NONNULL_END`

Reply:
Fixed